#47 Перенос обновлений с Beta на Alpha сервер

MysticalFaceLesS opened the pull request:
## Описание / Что этот PR делает

[Перенос №46] - (https://github.com/CeladonSS13/Shiptest/pull/1914)

Последний хотфикс на №46.2  - (https://github.com/CeladonSS13/Shiptest/pull/1935)

thearbiber committed:
Hermit gun stuff, part 1 (#5018)

<!-- Write **BELOW** The Headers and **ABOVE** The comments else it may
not be viewable. -->
<!-- You can view Contributing.MD for a detailed description of the pull
request process. -->

## About The Pull Request
resprites some hermit guns, (the rifle and zip pistol) removes the
safeties of said guns, gives them an ORGANIZED FOLDER, and reworks the
zip pistol to be more unique, more of an actual makeshift gun. (It's
"Full Auto" and still very dogass, outcompeted by every pistol)

![image](https://github.com/user-attachments/assets/1f9cba73-d297-4240-8cd4-eb988c4e6e2f)

## Why It's Good For The Game

these sprites were ancient, and the organizing of guns was awful here.
Hopefully should make adding more stuff in the future easier, and
something nice to look at?

## Changelog

:cl:
add: resprites the hermit rifle and zip pistol
balance: reworks the zip pistol, removes safeties off the zip pistol and
hermit rifle
/:cl:

<!-- Both :cl:'s are required for the changelog to work! You can put
your name to the right of the first :cl: if you want to overwrite your
GitHub username as author ingame. -->
<!-- You can use multiple of the same prefix (they're only used for the
icon ingame) and delete the unneeded ones. Despite some of the tags,
changelogs should generally represent how a player might be affected by
the changes rather than a summary of the PR's contents. -->

Erikafox committed:
Update Cirrus Crew Quarters (#5075)

## About The Pull Request
I thought what I mapped for the Cirrus's Crew space was a little
lacking, and put in to improve it.
Notably: Command staff have private quarters
The dorm has a gay little couch
Bathroom expanded

<img width="1600" height="800" alt="StrongDMM-2025-07-15 17 44 58"
src="https://github.com/user-attachments/assets/30351123-8698-435c-99fe-b2be02407339"
/>
<img width="1600" height="800" alt="StrongDMM-2025-07-15 17 45 16"
src="https://github.com/user-attachments/assets/94ed9301-2f45-4a74-9eb7-f07042217f8b"
/>



## Changelog

:cl:
add: remodels the fore of the cirrus
add: half-stair decal
fix: you can now map specific coffee cartridges
/:cl:

Erikafox committed:
restore minutebar fluff (#5098)

## Changelog

:cl:
fix: minute bar fluff un-regressed
/:cl:

Erikafox committed:
remove minefields (#5091)

## About The Pull Request



![dreamseeker_i9zFEiZAXZ](https://github.com/user-attachments/assets/46160178-29fd-402c-8a97-0a68a158e44f)

this has been my admin log since people landed on planets with
minefields

## Changelog

:cl:
del: random minefields
/:cl:

Erikafox committed:
fixes holocall from undefined (#5090)

:cl:
fix: holocalls now report who's calling
/:cl:

SapphicOverload committed:
Fixes being able to understand sign language without knowing sign language + refactors sign language to be an actual language (#4814)

<!-- Write **BELOW** The Headers and **ABOVE** The comments else it may
not be viewable. -->
<!-- You can view Contributing.MD for a detailed description of the pull
request process. -->

Understanding sign language now actually requires knowing sign language.

Sign language has also been refactored into an actual language, and
adding other non-spoken languages is now supported.

fix

:cl:
fix: Fixed being able to understand sign language without knowing sign
language
code: Refactors sign language into an actual language
/:cl:

<!-- Both :cl:'s are required for the changelog to work! You can put
your name to the right of the first :cl: if you want to overwrite your
GitHub username as author ingame. -->
<!-- You can use multiple of the same prefix (they're only used for the
icon ingame) and delete the unneeded ones. Despite some of the tags,
changelogs should generally represent how a player might be affected by
the changes rather than a summary of the PR's contents. -->

---------

Co-authored-by: Theos <theubernyan@gmail.com>
Co-authored-by: FalloutFalcon <86381784+FalloutFalcon@users.noreply.github.com>

generalthrax committed:
Onehalf Tweaks (#5067)

## About The Pull Request

- Makes the filled rooms fill naturally using a normal gas pump instead
of a volume
- Reduced level of plasma on already existing floors
- Added some more scribbled in crayon to the engineering room
- Removed two catwalks that annoyed me
- Added in some O- blood and an IV drip to medbay

## Why It's Good For The Game

Some stuff I forgot and too much plasma

## Changelog

:cl:
balance: Decreased amount of plasma in flooded rooms in Onehalf and
added some misc tweaks like blood packs
/:cl:

moffball19 committed:
NSSV Kiwi-class Prospector (#5056)

<!-- Write **BELOW** The Headers and **ABOVE** The comments else it may
not be viewable. -->
<!-- You can view Contributing.MD for a detailed description of the pull
request process. -->

## About The Pull Request

Adds the Kiwi-class Prospector, an N+S Logistics vessel centered around
mining, drilling, and exploration. Also changes the behaviour of the
furnace processing consoles to link with the furnace based on a range
variable rather than needing to be directly adjacent. Shouldn't have any
effect on already existent working setups (thank you @Gristlebee for
code help). Also adds a few new outfits to spawn every role unequipped,
like my previous colossus remap.

> "A common sight in N+S's Frontier fleet, the Kiwi-class acts as the
company's mainstay prospecting vessel. Outfitted with a heavy-duty
mining drill and armed mining team to ensure it's continued operation,
these vessels are built to locate and capitalize on valuable
near-surface mineral deposits. Boasting moderate cargo capacity, dual
ore refineries, and comfortable (if cramped) crew cabins, Kiwis are
designed to be self-sufficient for some time and tend to outcompete
smaller, independent mining ventures."
![kiwi
(ingame)](https://github.com/user-attachments/assets/d925c038-6936-43cd-9ef2-a3509352a3f0)

Crew Complement:
* Supply Director (Captain Analogue)
* Lead Miner (2ic)
* Miner (x3)
* Paramedic
* Engineer
* Cargo Technician (Ripley Operator)
* Assistant

<details>
<summary>StrongDMM Screenshots</summary>


![kiwi](https://github.com/user-attachments/assets/46952f01-2583-4920-bfc4-853f68d388ed)

![kiwi
(areas)](https://github.com/user-attachments/assets/b95d8d28-7f59-4543-91c1-766547afafb8)
</details>

## Why It's Good For The Game

Our beloved Corporate Cartel still lacks in-game representation
appropriate for their size and significance in-lore. N+S is called "the
galaxy's largest logistics company", but in-game only has a single cargo
freighter in their roster. This PR aims to diversify their lineup and
bring more players to the faction.
![kiwi
burrow](https://github.com/user-attachments/assets/a793e2d9-668e-47ad-9c46-4455c9d99606)

## Changelog

:cl:
add: Kiwi-class Prospector, a mining vessel operated by N+S Logistics
code: Processing consoles now use a range distance variable to detect
furnaces.
/:cl:

<!-- Both :cl:'s are required for the changelog to work! You can put
your name to the right of the first :cl: if you want to overwrite your
GitHub username as author ingame. -->
<!-- You can use multiple of the same prefix (they're only used for the
icon ingame) and delete the unneeded ones. Despite some of the tags,
changelogs should generally represent how a player might be affected by
the changes rather than a summary of the PR's contents. -->

generalthrax committed:
remove deepthroat (#5112)

## About The Pull Request

removes deepthroat

## Why It's Good For The Game

its back

## Changelog

:cl:
del: removed deepthroat
/:cl:

thearbiber committed:
Ramzi Assets (in 2021) (#5080)

<!-- Write **BELOW** The Headers and **ABOVE** The comments else it may
not be viewable. -->
<!-- You can view Contributing.MD for a detailed description of the pull
request process. -->
Adds ramzi clothing to the game, places them in various maps so they
arent unused.
<img width="973" height="642" alt="image"
src="https://github.com/user-attachments/assets/2a999b22-53f1-40dc-bb64-e51c1be48052"
/>
<img width="546" height="391" alt="image"
src="https://github.com/user-attachments/assets/8ba23c01-0f8f-4f03-b037-557589779f9c"
/>

pirates should be identifiable, not using old ass assets that look meh.
Believe this keeps in the spirit of "Old and dated ICW Clothes" even if
theyre called ramzi clothing (which is because pirates should be
identifiable)

:cl:
add: Various Ramzi Clique Clothing
/:cl:

<!-- Both :cl:'s are required for the changelog to work! You can put
your name to the right of the first :cl: if you want to overwrite your
GitHub username as author ingame. -->
<!-- You can use multiple of the same prefix (they're only used for the
icon ingame) and delete the unneeded ones. Despite some of the tags,
changelogs should generally represent how a player might be affected by
the changes rather than a summary of the PR's contents. -->

generalthrax committed:
Brings back the Oldskipper + Makes it Hardline (#5081)

## About The Pull Request

Reflavours the Mudskipper to being Hardline instead of Miskilamo

<img width="992" height="544" alt="image"
src="https://github.com/user-attachments/assets/e0dff765-ed77-4595-a5a6-9c9fd1a26da8"
/>

<img width="1141" height="669" alt="image"
src="https://github.com/user-attachments/assets/31ba0fa8-4999-46f6-b1aa-78c783a4a30b"
/>

<img width="992" height="544" alt="image"
src="https://github.com/user-attachments/assets/1077b369-b4d0-402d-a773-5a930d742b46"
/>

Other changes:
- Slightly changed desc
- Added anti-rad foam + backpack tank for it because its cool
- Added very basic ingredients and changed the kitchen to accommodate
more table space and utensils
- Moved weapons storage into the hallway
- Moved welding tank to Not be in front of the recharger and into
engineering
- Added new captain clothes to the bridge and moved the broad intercom
out of the bridge
- Adjusted cargo bay layout to keep it more open
- Switched some light tubes to bulbs to conserve power to run at 40kw
without compromising APCs
- Replaced utility suits with overalls
- Bunks
- Notice board in cryo
- Soap and towels in the bathroom
- Shuffled some things around to fit a fax and photocopier into the
office
- There shall be no Grimy Tile on my beautiful earth

- Changed all indie engineer outfits to no longer start with toolbelts
and insulated gloves. That stuffs mapped into lockers for you to put on
after getting out of cryo

## Why It's Good For The Game

Never was very fond of the Miskilamo-ing of the Mudskipper. I tried to
make it work with a few follow-up prs after it dropped but I still don't
like it. Mudskippers pretty geared and well-equipped for salvaging which
keeps it apart from the shittier Miskie ships anyways. And Hardline
doesn't have a small ship yet.

Couple other misc changes are just to use some new stuff and very basic
supplies for 506 FSC living.

## Changelog

:cl:
add: Reflavoured the Mudskipper to be a Hardline vessel. Many changes
including new walls, decal-style, and shuffling around of equipment and
machinery
add: Indie engineers now start with olive overalls
balance: Indie engineer roles no longer start with toolbelts and
insulated gloves. Both of these now spawn in engineer's closets.
/:cl:

SomeguyManperson committed:
fix #4362 (#5106)

<!-- Write **BELOW** The Headers and **ABOVE** The comments else it may
not be viewable. -->
<!-- You can view Contributing.MD for a detailed description of the pull
request process. -->

## About The Pull Request

fixes #4362

## Why It's Good For The Game

lol

## Changelog

:cl:
fix: you can no longer telekinetically execute people
fix: dual wielding automatic firing guns will no longer force you to
magdump both of the guns in an unstoppable rage
/:cl:

<!-- Both :cl:'s are required for the changelog to work! You can put
your name to the right of the first :cl: if you want to overwrite your
GitHub username as author ingame. -->
<!-- You can use multiple of the same prefix (they're only used for the
icon ingame) and delete the unneeded ones. Despite some of the tags,
changelogs should generally represent how a player might be affected by
the changes rather than a summary of the PR's contents. -->

---------

Signed-off-by: Theos <theubernyan@gmail.com>

SomeguyManperson committed:
fix #4713 (#5108)

<!-- Write **BELOW** The Headers and **ABOVE** The comments else it may
not be viewable. -->
<!-- You can view Contributing.MD for a detailed description of the pull
request process. -->

## About The Pull Request

fixes #4713

## Why It's Good For The Game

<!-- Please add a short description of why you think these changes would
benefit the game. If you can't justify it in words, it might not be
worth adding. -->

## Changelog

:cl:
fix: you can no longer get infinite attachment slots by pretending to
remove existing attachments so the gun thinks it can hold more
/:cl:

<!-- Both :cl:'s are required for the changelog to work! You can put
your name to the right of the first :cl: if you want to overwrite your
GitHub username as author ingame. -->
<!-- You can use multiple of the same prefix (they're only used for the
icon ingame) and delete the unneeded ones. Despite some of the tags,
changelogs should generally represent how a player might be affected by
the changes rather than a summary of the PR's contents. -->

Signed-off-by: Theos <theubernyan@gmail.com>

zimon9 committed:
Allows bodycameras to be worn around the neck (#5110)

<!-- Write **BELOW** The Headers and **ABOVE** The comments else it may
not be viewable. -->
<!-- You can view Contributing.MD for a detailed description of the pull
request process. -->

## About The Pull Request

A common complaint about bodycameras has been that they're a little too
restrictive in terms of where they can be placed on a person and still
function. A rework of bodycameras is planned, but in the meantime, this
should help alleviate the issue -- though only slightly.

## Why It's Good For The Game

Bodycameras are too restrictive, and this can provide a tiny bit of QOL.

## Changelog

:cl:
balance: allows bodycameras to be worn around the neck
/:cl:

<!-- Both :cl:'s are required for the changelog to work! You can put
your name to the right of the first :cl: if you want to overwrite your
GitHub username as author ingame. -->
<!-- You can use multiple of the same prefix (they're only used for the
icon ingame) and delete the unneeded ones. Despite some of the tags,
changelogs should generally represent how a player might be affected by
the changes rather than a summary of the PR's contents. -->

generalthrax committed:
Scope Audit (#5086)

## About The Pull Request

Locks scopes and long scopes from most weapons
- Assault rifles can no longer mount either scope because of high
firerate and high accuracy
- Accelerator Laser Cannons and E50s can no longer mount either scope
because of hitscan / nine zillion burn projectile
- Reduced zoom amount from medium scopes to give you only about 4 tiles
of extra visibility
- Long scopes are locked to the Illestren, lever actions like the
Absolution, and the Beacon (we have sniper rifle at home)
- Prevents the Hydra DMR from using Extended and Casket mags. This is
kind of out of scope (hah) but I may as well hit something that's been
bothering me with one of the DMRs in the DMR balance pr
- Snipers (and longer ranged ones like the Taipan) maintain their zoom
- DMRs maintain their zoom
- Lets a few weird guns keep scopes like shotguns and pistols keep
scopes since they're inaccurate so you can use em for scouting or some
shit idk

## Why It's Good For The Game

They're pretty unfair with how the game is with how much visibility you
have on your screen by default. This mostly hits attachments since our
weapons that actually have inbuilt scopes are balanced around low rate
of fire and low magazine sizes

Hydra DMR is kind of unrelated but this locks it to not be too braindead

## Changelog

:cl:
balance: Prevents most automatic guns from mounting scopes (along with a
few more like the E50 and Accelerator laser cannon)
balance: Locks the long scope to the Illestren, lever-actions, and
Beacon
balance: Decreased medium scope zoom to give you only about 4 tiles of
extra visibility
balance: Hydra DMR can no longer mount extended or casket mags
/:cl:

Erikafox committed:
remove panacea (#5121)

:cl:
del: Panacea
/:cl:

Erikafox committed:
fix ssg not taking scope (#5123)

:cl:
fix: f4 can take scope again. whoops.
/:cl:

generalthrax committed:
fixes invictus mag cargo (#5120)

## About The Pull Request

forgot to actually put the mag up in cargo

## Why It's Good For The Game

good

## Changelog

:cl:
fix: Invictus magazine is now in cargo
/:cl:

---------

Signed-off-by: generalthrax <139387950+generalthrax@users.noreply.github.com>
Co-authored-by: Erika Fox <94164348+Erikafox@users.noreply.github.com>

Anticept committed:
Fixes build.js so that it properly prepends #define CBS, suppressing compiler warnings (#5033)

@MarkSuckerberg 

I wanted to do more than this but it turns out that _compiler_options.dm
is too late in the dme file order, so I can't move FULL_INIT out of
_basemap.dm without more sweeping changes and that's just out of scope.

So here's the CBS fix (and a tweak on the FULL INIT comment)

Tested on frontier starworks with TGS v6.17.0. This will cleanly prepend

```
#define TGS
#define CBT
```

to the DME before building, so you no longer need to use TGS's
codemodifications menu to add `#define CBT`, the buildchain does it
properly now.

Thanks to @FalloutFalcon for finding the `join` solution. The function
was adding commas without it.

## Changelog

:cl: Anticept, FalloutFalcon 
server: the build chain used in TGS has been fixed so that "#define CBS"
is properly prepended to shiptest.dme, and suppresses compiler warnings
regarding building the dme directly.
/:cl:

Co-authored-by: Mark Suckerberg <mark@suckerberg.gay>

SapphicOverload committed:
Fixes manifest and join menu UIs not updating properly (#5019)

:cl:
fix: Fixed manifest and join menu UIs not updating properly
fix: Fixed certain text characters breaking memos
/:cl:

thearbiber committed:
Vigilitas deepstorage - New space ruin (also shoots bigderelict1) (#5076)

<!-- Write **BELOW** The Headers and **ABOVE** The comments else it may
not be viewable. -->
<!-- You can view Contributing.MD for a detailed description of the pull
request process. -->

## About The Pull Request

Adds in a new space ruin. Fronties are shooting a VI storage plant. How
unfortunate. also kills bigderelict because it sucks ass
Notable loot is guns, drugs, and Something Special
<img width="2560" height="2560" alt="image"
src="https://github.com/user-attachments/assets/f7df5059-9bf9-4e86-a806-92829debdf29"
/>
<img width="2560" height="2560" alt="image"
src="https://github.com/user-attachments/assets/4f85ece9-61d3-4530-bc48-49d3ad6e82cf"
/>

## Why It's Good For The Game
bigderelict sucked. As for the ruin more ruins+shouldnt be a megahell
one even if it looks like it at a glance

## Changelog

:cl:
add: Vigilitas Deepstorage, space ruin
del: Bigderelict1
/:cl:

<!-- Both :cl:'s are required for the changelog to work! You can put
your name to the right of the first :cl: if you want to overwrite your
GitHub username as author ingame. -->
<!-- You can use multiple of the same prefix (they're only used for the
icon ingame) and delete the unneeded ones. Despite some of the tags,
changelogs should generally represent how a player might be affected by
the changes rather than a summary of the PR's contents. -->

thearbiber committed:
Adds the Venetia-class Frontier Yacht (#5109)

<!-- Write **BELOW** The Headers and **ABOVE** The comments else it may
not be viewable. -->
<!-- You can view Contributing.MD for a detailed description of the pull
request process. -->

## About The Pull Request
Noticed we were lacking "comfy big ship" vibe for a bit, and figured
that one that can do gameplay shit would be fun. Starts with 1 captain,
1 2ic, 2 Inteq enforcers, 2 scrappers, and four passengers.
<img width="928" height="864" alt="image"
src="https://github.com/user-attachments/assets/93be0c9d-3d61-4b1b-8469-80470371ba0e"
/>
<img width="928" height="864" alt="image"
src="https://github.com/user-attachments/assets/d626f37b-fcd2-47a9-b0f4-66a0dd5597e6"
/>
<img width="1139" height="961" alt="image"
src="https://github.com/user-attachments/assets/3689d1a4-d6ee-4e98-a34b-17f4bab8fb6c"
/>

## Why It's Good For The Game
Think a more relaxing ship would be good for the game. This one shouldnt
be sitting in port all day either, believe i gave it enough reason to go
out and still Do Things while also having a mix of relaxing inside. The
part inteq compliment should be interesting aswell.

## Changelog

:cl:
add: Venetia-class Superyacht
/:cl:

<!-- Both :cl:'s are required for the changelog to work! You can put
your name to the right of the first :cl: if you want to overwrite your
GitHub username as author ingame. -->
<!-- You can use multiple of the same prefix (they're only used for the
icon ingame) and delete the unneeded ones. Despite some of the tags,
changelogs should generally represent how a player might be affected by
the changes rather than a summary of the PR's contents. -->

generalthrax committed:
fixes onehalfs holofields (#5141)

## About The Pull Request

forgor

## Why It's Good For The Game

fix

## Changelog

:cl:
fix: Onehalf's holofields are now anchored
/:cl:

Ical92 committed:
Removes shuttles from Autowiki Shiptable generations (#5142)

<!-- Write **BELOW** The Headers and **ABOVE** The comments else it may
not be viewable. -->
<!-- You can view Contributing.MD for a detailed description of the pull
request process. -->

## About The Pull Request

<!-- Describe The Pull Request. Please be sure every change is
documented or this can delay review and even discourage maintainers from
merging your PR! -->

Updates `code\modules\autowiki\pages\shiptable.dm` so shuttles aren't
shown on the [autowiki
shiptable](https://shiptest.net/wiki/Template:Autowiki/Content/ShipTable).
## Why It's Good For The Game
Getting the autowiki shiptable to a more usable state, no player-facing
changes.
<!-- Please add a short description of why you think these changes would
benefit the game. If you can't justify it in words, it might not be
worth adding. -->

## Changelog

:cl:
/:cl:

<!-- Both :cl:'s are required for the changelog to work! You can put
your name to the right of the first :cl: if you want to overwrite your
GitHub username as author ingame. -->
<!-- You can use multiple of the same prefix (they're only used for the
icon ingame) and delete the unneeded ones. Despite some of the tags,
changelogs should generally represent how a player might be affected by
the changes rather than a summary of the PR's contents. -->

MarkSuckerberg committed:
516 TGUI Hotfix (#5131)

Ports most of tgstation/tgstation#82527

> Apparently you cant just stuff the byond helper functions into an
external js file, but if you do, byond won't even let you know its a
problem until the servers crash and you have to run `bin/clean` just to
unbork your entire repo
>
> This reimplements the changes from tgstation/tgstation PR 82473
without:
> - moving the byond helper functions externally
> - causing a tooltip render issue in panel
> 
> 516 prep (again this time)

## Changelog

:cl: jlsnow301
fix: Gets TGUI <i>FUNCTIONAL</i>, but not necessarily smooth on 516.
There's still going to be a lot of issues, but at least it'll be
playable.
/:cl:

<!-- Both :cl:'s are required for the changelog to work! You can put
your name to the right of the first :cl: if you want to overwrite your
GitHub username as author ingame. -->
<!-- You can use multiple of the same prefix (they're only used for the
icon ingame) and delete the unneeded ones. Despite some of the tags,
changelogs should generally represent how a player might be affected by
the changes rather than a summary of the PR's contents. -->

---------

Co-authored-by: Jerm <jlsnow301@pm.me>
Co-authored-by: Lucy <lucy@absolucy.moe>
Co-authored-by: Aylong <69762909+AyIong@users.noreply.github.com>
Co-authored-by: AnturK <4047233+anturk@users.noreply.github.com>
Co-authored-by: harry <me@harryob.live>
Co-authored-by: harryob <55142896+harryob@users.noreply.github.com>
Co-authored-by: itsmeow <itsmeow@itsmeow.dev>

Erikafox committed:
fix laughs not happening (#5115)

## About The Pull Request
whoops

## Changelog

:cl:
fix: you can live laugh love again
/:cl:

Aquidu committed:
Makes some adjustments to the Harrier. (#5071)

<!-- Write **BELOW** The Headers and **ABOVE** The comments else it may
not be viewable. -->
<!-- You can view Contributing.MD for a detailed description of the pull
request process. -->

## About The Pull Request
Makes a variety of minor additions and changes to the Harrier-class,
without changing any core design features of it.
Complete list of changes:
<details><summary>Cargo:</summary>
<p>

- Swapped the miner's engineering goggles to just mesons.

- Added a spare jumpsuit for the miner.

</p></details>
<details><summary>Canteen:</summary><p>

- Added card games, dice, plushies, and moved the guitar from
Engineering here.

- Added a bookshelf

- Added universal enzyme to the kitchen for making cheese and other
recipes.

- Added several fruits and vegtables as well as some misc supplies to
the kitchen cabinet to feed the many crewmembers.

</p></details>
<details><summary>Crew Quarters:</summary><p>

- Added pens, crayons, and a paper bin for recreational writing.

- Added an easel and art closet for painting.

- Added spare Assistant clothes and shoes to the laundry cart.

</p></details>
<details><summary>Security Office:</summary><p>

- Replaced the Mini Energy gun with an E-TAR, as well as gave it a spare
cell(the mini-e only had one cell).

</p></details>
<details><summary>Bridge:</summary><p>

- Added an extra weapon power cell to the Captain's locker for the
energy rifle in the gun locker(same story as mini-e).

- Fixed one window grille of the Bridge not being electrified.

- Added a box of spare PDAs to the FO's locker, as well as a PDA
painter.

- Added some decor to the FO's office to make it match the Captain's.

- Added a trans flag next to the fax(VERY IMPORTANT)

- Replaced the coffee vendor with a cheap coffee machine.

- Swapped the placement of the alert console and sensors console.

</p></details>
<details><summary>Cryogenics:</summary><p>

- Added radio closet.

</p></details>
<details><summary>General:</summary><p>

- Changes shuttle window spawners to grille+shuttle windows.

- Adds fitting cleanables throughout the ship.

- Removed role spawn locations.

- Added a object of a filled glass of Bridge Bunny.

- Fixed bug where a flock of birds would descend on the ship and cause
chaos:


![dreamseeker_e9bVFLEsgC](https://github.com/user-attachments/assets/87c4785a-9fed-4108-b88a-3bd5f749bbc3)
</p></details>
<!-- Describe The Pull Request. Please be sure every change is
documented or this can delay review and even discourage maintainers from
merging your PR! -->

## Why It's Good For The Game
All the above changes have been done as a result of my extended time
captaining the Harrier in-game, and the issues I have run into. Each
change here I believe will help the ship toward a more enjoyable
experience for players, without invalidating or changing the core
gameplay of the ship.

Cleanables, toys, and art:
The crew of the Harrier is very large, as such the entire crew cannot
participate in a mission the same way the entire crew of a Kilo would,
as there simply is not enough things for them to all take part in a
meaningful fashion, or without being left behind. This is okay though,
as long as staying behind does not mean the players are denied the
ability to interact with the players. The addition of these will
hopefully provide more things for players to do during downtime or if
they need to stay at the ship.

Extra clothes:
Clothes can be destroyed for a number of reasons, and since they aren't
purchasable it's good to have spares. The Harrier already had some, but
I have added the missing ones for consistency.

Extra food:
The Harrier can have up to 13 crewmembers total, and the current pantry
barely provides enough for everyone to be fed out of Cryo, let alone fed
throughout the day(Not even factoring some crewmembers may have diet
restrictions). Other ships with far less crew have been granted massive
pantries, such as the Corona, Ivory, and Nimbus. This PR will bring the
Harrier's stores in line with them more, while still limiting the amount
to give incentive for ingredient collection or purchasing later. I
believe the Harrier should also have a strong kitchen, since it, aside
from the Sunskipper, is the only ship with a dedicated chef.

Weapons:
The Harrier is among the least-equipped ships in the game, despite its
large crew. While I understand that it is a civilian ship, it's very,
very hard to get the ball rolling on the Harrier compared to other
ships, due in part to it's miniscule armory. I added extra power cells
out of consistency, since every gun purchase comes with two magazines, I
thought it would be only fair to give a spare cell to both the energy
rifle and E-TAR. I replaced the energy pistol with the E-TAR because
anyone who's used the energy pistol will tell you that thing does not
count as a real weapon. I didn't want to go overboard with something
like a E-SG or Hades, so I think the E-TAR is a nice boon while not
being too upsetting.

If you would like me to elaborate on any specific change, please comment
below.
<!-- Please add a short description of why you think these changes would
benefit the game. If you can't justify it in words, it might not be
worth adding. -->

## Changelog

:cl:
add: Adds a wide variety of changes to the Harrier
code: Added a glass full of Bridge Bunny as a placeable object
/:cl:

<!-- Both :cl:'s are required for the changelog to work! You can put
your name to the right of the first :cl: if you want to overwrite your
GitHub username as author ingame. -->
<!-- You can use multiple of the same prefix (they're only used for the
icon ingame) and delete the unneeded ones. Despite some of the tags,
changelogs should generally represent how a player might be affected by
the changes rather than a summary of the PR's contents. -->

generalthrax committed:
Removes the F4's Pre-Attached Scope and makes it Integrated (#5144)

## About The Pull Request

<img width="251" height="177" alt="image"
src="https://github.com/user-attachments/assets/e6eecc94-1aef-43e5-8398-544d0f41375a"
/>

Pretty much makes it not an attachment and part of the gun. Girl
unrelated. Thanks to moffball for the sprites @moffball19

## Why It's Good For The Game

It got hit by the scope audit when I didn't want it to.

## Changelog

:cl: generalthrax, moffball19
balance: F4 and variants no longer have a scope attachment and instead
have an integrated scope
/:cl:

generalthrax committed:
removes sand world gibtonite (#5145)

## Why It's Good For The Game

forgot in the last gibtonite cut

## Changelog

:cl:
del: removed gibtonite from sand planet randomgen
/:cl:

thearbiber committed:
Add: Респрайты одежды из (shiptest-ss13#5140)

KOCMODECAHTHUK committed:
GITHUB: Добавление проверок на 2 ветку от куда идут обновления

Gristlebee committed:
Suit Storage Locks now respect access (#5136)

<!-- Write **BELOW** The Headers and **ABOVE** The comments else it may
not be viewable. -->
<!-- You can view Contributing.MD for a detailed description of the pull
request process. -->

## About The Pull Request

<!-- Describe The Pull Request. Please be sure every change is
documented or this can delay review and even discourage maintainers from
merging your PR! -->

Title. SSU locks can now respect access. You can adjust an SSU's access
by installing an airlock circuit. Installed circuits can be removed with
a crowbar. The circuit must be uninstalled before deconstruction.

Adds a wire to hack the lock.

## Why It's Good For The Game

<!-- Please add a short description of why you think these changes would
benefit the game. If you can't justify it in words, it might not be
worth adding. -->

The lock on SSUs were literally useless since they didn't respect
access, and the only edge case they did something was when the power
happened to be out. This should make them actually usable.

## Changelog

:cl:
add: Suit Storage Unit locks respect access
/:cl:

<!-- Both :cl:'s are required for the changelog to work! You can put
your name to the right of the first :cl: if you want to overwrite your
GitHub username as author ingame. -->
<!-- You can use multiple of the same prefix (they're only used for the
icon ingame) and delete the unneeded ones. Despite some of the tags,
changelogs should generally represent how a player might be affected by
the changes rather than a summary of the PR's contents. -->

Gristlebee committed:
EMPs can longer randomize locker access (#5124)

<!-- Write **BELOW** The Headers and **ABOVE** The comments else it may
not be viewable. -->
<!-- You can view Contributing.MD for a detailed description of the pull
request process. -->

## About The Pull Request

<!-- Describe The Pull Request. Please be sure every change is
documented or this can delay review and even discourage maintainers from
merging your PR! -->

## Why It's Good For The Game

<!-- Please add a short description of why you think these changes would
benefit the game. If you can't justify it in words, it might not be
worth adding. -->

Your clothing locker shouldnt require gateway access to open.

## Changelog

:cl:
del: EMPs randomize locker access
/:cl:

<!-- Both :cl:'s are required for the changelog to work! You can put
your name to the right of the first :cl: if you want to overwrite your
GitHub username as author ingame. -->
<!-- You can use multiple of the same prefix (they're only used for the
icon ingame) and delete the unneeded ones. Despite some of the tags,
changelogs should generally represent how a player might be affected by
the changes rather than a summary of the PR's contents. -->

ritorizo committed:
Make it so suit pockets are openable again (#5117)

## About The Pull Request

Some suits can be opened and closed, and also have pockets.
Both accessing the pockets and opening/closing is assigned to 
AltClick. 
Right now alt clicking makes it so you just toggle the suit making it's
inventory inaccessible.
With this pr AltClicking a suit both open it's inventory and toggles it.

## Why It's Good For The Game

Having access to pockets is good and having the suits be toggled when
you do so gives a visual indicator that the person wearing is looking
trough it's suit.

## Changelog

:cl:
fix: The inventory of coats and labcoat is accessible again.
:cl:

<!-- Both :cl:'s are required for the changelog to work! You can put
your name to the right of the first :cl: if you want to overwrite your
GitHub username as author ingame. -->
<!-- You can use multiple of the same prefix (they're only used for the
icon ingame) and delete the unneeded ones. Despite some of the tags,
changelogs should generally represent how a player might be affected by
the changes rather than a summary of the PR's contents. -->

Co-authored-by: ritorizo <ritorizo@localhost>

Erikafox committed:
removes spraypainting clothes (#5103)

## Changelog

:cl:
del: spraypainting clothing
/:cl:

Erikafox committed:
Remove purchasing concealed weapons bays (#5101)

## About The Pull Request

you can no longer purchase concealed weapons bays on the black market

## Changelog

:cl:
balance: you can no longer purchase concealed weapons bays on the black
market
/:cl

Erikafox committed:
Nerf Solaris (#5102)

## Changelog

:cl:
balance: Solaris scope removed, Solaris power cost increased, Solaris
cycle time increased
/:cl:

Erikafox committed:
remove fax spam (#5126)

## About The Pull Request

this adds nothing bro

## Changelog

:cl:
del: fax spam
/:cl:

KOCMODECAHTHUK committed:
UPDATE-SYNC: Newfood (The Rest of the Repaths) (#1906)

* Newfood (The Rest of the Repaths) (#4886)
Why the fuck was oatmeal a salad
<!-- Describe The Pull Request. Please be sure every change is
documented or this can delay review and even discourage maintainers from
merging your PR! -->

:cl:
fix: rest of the food repathing, report any visible issues
/:cl:

<!-- Both :cl:'s are required for the changelog to work! You can put
your name to the right of the first :cl: if you want to overwrite your
GitHub username as author ingame. -->
<!-- You can use multiple of the same prefix (they're only used for the
icon ingame) and delete the unneeded ones. Despite some of the tags,
changelogs should generally represent how a player might be affected by
the changes rather than a summary of the PR's contents. -->

* [MANUAL-FIX]

* [AUTO-FIX]

* [MAP-FIX]

---------

Co-authored-by: thgvr <81882910+thgvr@users.noreply.github.com>

KOCMODECAHTHUK committed:
RETURN: Old Version Mine Field

SapphicOverload committed:
Prevents ion storms from softlocking ships (#4996)

<!-- Write **BELOW** The Headers and **ABOVE** The comments else it may
not be viewable. -->
<!-- You can view Contributing.MD for a detailed description of the pull
request process. -->

## About The Pull Request

Significantly reduces the time ion storms disable APCs for, preventing
them from softlocking ships. Considering ion storms are the only source
of EMPs players are likely to ever encounter, I felt this was the right
solution to the problem without removing them from the game entirely.

Alternative to #4993

## Why It's Good For The Game


https://github.com/user-attachments/assets/8927fe30-3567-4849-a284-6a299cda96a5

## Changelog

:cl:
balance: Significantly reduced the time ion storms disable APCs for,
preventing softlocks
/:cl:

<!-- Both :cl:'s are required for the changelog to work! You can put
your name to the right of the first :cl: if you want to overwrite your
GitHub username as author ingame. -->
<!-- You can use multiple of the same prefix (they're only used for the
icon ingame) and delete the unneeded ones. Despite some of the tags,
changelogs should generally represent how a player might be affected by
the changes rather than a summary of the PR's contents. -->

Erikafox committed:
Breach Mining (#5057)

<!-- Write **BELOW** The Headers and **ABOVE** The comments else it may
not be viewable. -->
<!-- You can view Contributing.MD for a detailed description of the pull
request process. -->
Changes how drilling works.

Drills now spawn "seismic disturbances" that spawn the monsters. They
stay in place for the duration of the "wave". After a certain amount of
waves you get a short break that is EXPLICITLY NOTIFIED

![dreamseeker_euQbH5FbEw](https://github.com/user-attachments/assets/a3bebab0-62c4-4b88-b63c-ef1818e1d075)

![dreamseeker_k03O0zdMKd](https://github.com/user-attachments/assets/95c8ce1f-7013-4cd9-a2aa-10c176fbc37e)

:cl:
add: Drills now spawn a spawner, which in turn spawns the monsters. It's
an innovative system really.
fix: ACTUALLY FIXED DRILL TURF PLACEMENT THIS TIME
/:cl:

KOCMODECAHTHUK committed:
fix: Мимы сломались, ну и похуй

KOCMODECAHTHUK committed:
del: Инкит содержимого коробки с жевачкой

KOCMODECAHTHUK committed:
tweak: Обновление правил путей к карманам у шлемов

KOCMODECAHTHUK committed:
GITHUB: Изменение в автоматизации под авто-чейнжлог (#1965)

MysticalFaceLesS committed:
Merge pull request #1941 from CeladonSS13/UPDATE-SYNC

UPDATE-SYNC: ОБНОВЛЕНИЕ с АПСТРИМОМ #18

KOCMODECAHTHUK committed:
fix: Invisible ammo box .308 (#1968)

PumPeyMey committed:
MAP: New Ramzi pirate ship (#1955)

* Add New ship Rondo

* some tweak and fix

Поменял двигатели на ионный из-за проблем с атмосом, провел проводку к голофанам, немноного подвигал фразы на обшивке, заменил синди-вендор с сигаретам на обычный

* More tweak

Замена дрипа на рамзи-дрип + дал более понятное название аутфитам которые используются на Рондо

* del cigpack syndicate

* fix json file

---------

Co-authored-by: Kenfarka <namor4ikm@gmail.com>

Asuraqami committed:
MAP: Замена Болида - Daggerfall (#1971)

* ADD: New ship - Daggerfall.

* Update nanotrasen_daggerfall.dmm

* Update nanotrasen_daggerfall.dmm

* фикс субшаттла

---------

Co-authored-by: Kenfarka <namor4ikm@gmail.com>

Kenfarka committed:
Update nanotrasen_bolide.json (#1972)

Cuildipie committed:
TWEAK: убираем дилей снятия/надевания всех рюкзаков и подтипов рюкзаков (#1975)

* TWEAK: убираем дилей снятия/надевания всех рюкзаков и подтипов рюкзаков
* tweak: На всякий случай убрал и у сачтелов нулевую задержку
* Поменял EDIT на REMOVE

Asuraqami committed:
MAP: Celestis buff (#1946)

* MAP: Celestis buff

* Update nanotrasen_celestis.json

* Update nanotrasen_celestis.dmm

* Update nanotrasen_celestis.dmm

* Update nanotrasen_celestis.dmm

* Update nanotrasen_celestis.dmm

Cuildipie committed:
MAP: Squab - экипировка персонала (#1974)

KOCMODECAHTHUK committed:
TWEAK: Сатчелы теперь не имеют меньший размер по сравнению с рюкзаками (#1976)

KOCMODECAHTHUK committed:
TWEAK: Автолат - Стопки кратные 15х и 30х (#1949)

* TWEAK: Автолат - Стопки кратные 15х и 30х
* comment: Стилистика кода

MrCat15352 committed:
REMOVE: Отключаем сокрытие хвостов риолам и таярам (#1978)

* Чиним плохой код риолов

* Отключаем модпак от системы

Mirag1993 committed:
ADD: Missing syndicate medical meka upgrade module (#1987)

Описания ПР точно не писала нейронка, я вам отвечаю!
- Added /obj/item/borg/upgrade/transform/meka/syndicate_medical
- Added corresponding design borg_upgrade_meka_syndicate_medical
- Added design to techweb node cyborg_upg_meka
- Fixes issue where syndicate medical borgs couldn't transform to meka body

PumPeyMey committed:
MAP: Fix some maps

KOCMODECAHTHUK committed:
REMOVE: Хуйня хуйни, на инициализацию магазинов у G36 (#1960)

KOCMODECAHTHUK committed:
RETURN: LEGEND PLASMA CUTTER (#1985)

Asuraqami committed:
MAP: All RND SHIP UP (#1982)

* MAP: All RND SHIP UP

* Update nanotrasen_squab.json

MysticalFaceLesS committed:
BALANCE: Изменение характеристик для Energy Halberd & Double-Bladed Energy Sword + New Shuttle Area Editor (#1956)

* melee update
* sound update
* new areaeditor (#1993)

KOCMODECAHTHUK committed:
FIX: Прическа риолов (#1948)

MysticalFaceLesS committed:
❤ Hello world! New a secret folder for Celadon

MysticalFaceLesS committed:
[SOUND] Change sound Prays

KOCMODECAHTHUK committed:
fix: Разные крафт категории (#1999)

Pelmenik123 committed:
FIX: Перехват контроля ИИ над игроком в симпл-мобах (#2008)

MrCat15352 committed:
Фикс рецепта святой воды (#2014)

KOCMODECAHTHUK committed:
BALANCE: Снижаем шансы и общую прочность щитам (#1958)

* BALANCE: Снижаем шансы и общую прочность щитам

* Ещё чутка снижаем параметры

* tweak: Баллистический щит

Signed-off-by: KOCMOHABT <25280520+KOCMODECAHTHUK@users.noreply.github.com>

---------

Signed-off-by: KOCMOHABT <25280520+KOCMODECAHTHUK@users.noreply.github.com>

KOCMODECAHTHUK committed:
FIX & TWEAK: Cleaving saw (#2012)

* fix: Звука на энерго-меч

* add: Добавлены звуки открытия/закрытия из BloodBorne

* spellcheck: Оформление

KOCMODECAHTHUK committed:
fix: Микро ошибки в коде (#2011)

Vairkharst reviewed (commented):
А когда на 516 перейдем?

KOCMODECAHTHUK committed:
ADD: Добавляет свиней в игру (#2010)

Pelmenik123 committed:
FIX: Фикс прилипания планет к кораблю (#2017)

* ПОСЛЕДНИЙ ЗАЕЗД

Mirag1993 committed:
Заголовок Чиним диски дизайнов: запись, очистка и печать снова работают стабильно (#2016)

Диски дизайнов снова корректно записывают выбранные чертежи через R&D консоль.
Можно очищать отдельный слот или весь диск; меню больше не пропадает.
Чережтежи не «перепрыгивают» между слотами после очистки.
Автолат снова видит чертежи на диске и печатает совместимые предметы.
Продвинутые диски (3/5/10 слотов) работают так же, по индексам слотов.
Старые «сломанные» диски автоматически нормализуются при открытии консоли.

KOCMODECAHTHUK committed:
ADD: Новые худы и правки в Аутфитах (#1986)

* add: Худы и правки в Аутфитах

* fix: Иконки в аутфите у СФ

* add modular

* add: Фикс спрайтов

* spellcheck: Оформление по мелочи

* add: Старый внешний вид хадов но с кубами в крит состоянии

* tweak: Медик пиратов становится парамедом

* tweak: Изменены пути к хадам

---------

Co-authored-by: MysticalFaceLesS <MrLamaScura@gmail.com>

KOCMODECAHTHUK committed:
ADD: Добавление в атлас респрайтов к ксеносам, и респрайт маски синдиката (отдаем честь старой анимации маски синди) (#1964)

* add: Правка масок для ксеносов

* add: Микромомент с изменением

* tweak: Различные правки

* маски общие

* add: Обновление спрайтов от асптрима

* fix: update file

* add: Очередное дополнение

* add: По мелочи ещё маски

* add: Бригада СФ

Cuildipie committed:
MAP: Delta - выводим из щитспавна с небольшим ребелансом (#1983)

* MAP: Meta - выводим из щитспавна с небольшим ребелансом

KOCMODECAHTHUK committed:
ADD: Инициализация ИНТЕРЕСНОГО (#2028)

* add: Инициализация интересного

* Mistake oops

Kenfarka committed:
MAP: Jungle Syndicate Remap (#1997)

* MAP: Jungle Syndicate Remap

Mirag1993 committed:
FIX: G36 attachments (#2026)

* Фикс аттачментов для G36
* tweak: Атачменты марки СФ

---------

Co-authored-by: KOCMOHABT <25280520+KOCMODECAHTHUK@users.noreply.github.com>

KOCMODECAHTHUK committed:
[MIRROR]: Поддержка кириллицы, подсветка чата и многое другое (#1984)

* Fixing the chat client (tgstation/tgstation#60920)
* Fixes paper text length not counting correctly for cyrillic characters (tgstation/tgstation#75021)
* tweak: Подсветка кирилицы в чате
* TGchat: Tweaks & Fixes
(tgstation/tgstation#81354)

Myakotka committed:
FIX: Reverse Supplypod - в саплай панели (#2019)

* фикс ревёрса в саплай панели
* add: Небольшие дополнения и правки

---------

Co-authored-by: Myakotka <Myakotka@users.noreply.github.com>
Co-authored-by: KOCMOHABT <25280520+KOCMODECAHTHUK@users.noreply.github.com>

Kenfarka committed:
MAP: Titanium windows on panacea (#2030)

* MAP: титановые окна на панацее

* Update syndicate_panacea.dmm

Asuraqami committed:
MAP: Tomahawk remapp, some tweak to daggerfall (#2004)

* MAP : Remapp Tomahawk, one tweak to Daggerfall

MysticalFaceLesS committed:
ADD: Discord Verify Update (#2032)

* ADD: Discord Verify Update
* Add module
